src/E1_coldflow/E1_coldflow.cpp

```diff
@@ -105,26 +100,25 @@ void loop() {
     int i = 0;
     while (RFSerial.available()) {
       command[i] = RFSerial.read();
-      Serial.print(command[i]);
       i++;
     }
```

Comment:
Do we not want to check if the first two bytes constitute a start packet command every time a new byte is received?

Reply:
I was doing that in the decode packet. But maybe it makes more sense to do that here. I can also then stop when I see the end bytes

Reply:
:+1: Yeah I think doing it here is better